ui/README.md

```diff
@@ -1,7 +1,33 @@
 # Contributor Guide for UI component
 
-<p style="text-align: center"><em>If you'll like to go to the main Meshery Contributor guide <a href="../CONTRIBUTING.md">click here</a></em></p>
+This guide is specific to the Meshery UI component and involves steps/methods one need to follow while working on issues related to Meshery UI.
 
-## Meshery UI doing server-side rendering of NextJS/React components.
+## How to run Meshery UI?
+Meshery UI can be used for different purposes, it can be for general use like testing out Meshery and it's functionalities or work on issues related to the Meshery UI.
 
-## Meshery UI uses billboard.js as the charting library.
+#### 1. General Usage:
+For general usage, one can run Meshery UI using Meshery's command client `mesheryctl`, by simply running the `mesheryctl system start` command.
+If you don't have the `mesheryctl` tool installed already, you can follow the [mesheryctl installation docs](https://docs.meshery.io/installation/mesheryctl) to install `mesheryctl` using various `package management` tools supported.
+
+#### 2. For development purporse:
+For development purpose, either of the below mentioned approach could be followed
+- Follow the procedure mentioned in Step 1 (General Usage) above, and start Meshery UI sever on the 9081 port, and login to Meshery UI using eihter of the providers mentioned on the login page. Then, to run a development server of Meshery UI, first install the dependencies using the `make setup-ui-libs` command, then build the UI package and export it using the command `make build-ui`, and finally execute `make run-ui-dev` to run the livereload-nodemon server on port 3000.
```

Comment:
@Nikhil-Ladha, does `make-setup-libs` need to be pulled out and identified as a one-time prerequisite command to execute irrespective of which method the developer uses?

Reply:
I think so, yes.